wake/demo.wake

```diff
@@ -104,3 +104,10 @@ publish vivadoVsrcHooks =
   makeBlackBoxHook "pio"          "{blockPIOSifiveRoot}/rtl/pio/pio.sv".makeSource,
   makeBlackBoxHook "loopback"     "{blockPIOSifiveRoot}/rtl/loopback/loopback.sv".makeSource,
   Nil
+
+
+# Documentation
+publish scribbleDirectories =  simplify "{here}/../docs/scribble", Nil
+
+# Build the pio document onlining document from scratch
+global def makePioDocument Unit = makeOnboardingDocument pioDUT
```

Comment:
this seems unnecessary

Reply:
I agree, especially after we introduced the second PIO configuration (pio16DUT). Removed in https://github.com/sifive/block-pio-sifive/pull/24/commits/1447daf5ab46533af58dbe4340ee303ccd944b8e.